packages/subgraph/test/validation/aggregateValidators.ts

```diff
@@ -53,7 +53,7 @@ export const validateATSEntity = (
         totalDeposit: expectedTotalDeposit,
     } = expectedATSData;
 
-    const calculatedCriticalTimestamp = calculateMaybeCriticalAtTimestamp(expectedATSData.updatedAtTimestamp, expectedBalanceUntilUpdatedAt, expectedTotalDeposit, expectedTotalNetFlowRate);
+    const calculatedCriticalTimestamp = calculateMaybeCriticalAtTimestamp(expectedATSData.updatedAtTimestamp, expectedBalanceUntilUpdatedAt, expectedTotalNetFlowRate);
```

Comment:
Just saying, it's not optimal to effectively use the implementation to assert this... but I'm not familiar with the tests and there might not be a much better way to do this.

Reply:
This is the pattern currently present in both the ethereum contracts/subgraph.
We have an initial local state, then we actually modify it (send txn), then we update our local state based on expected behavior logic and compare the local state to the actual state after the modification.
We have expected behavior logic which we use to calculate the information again, this relies on the assumption that our expected behavior logic is correct.